src/uct/base/uct_iface.c

```diff
@@ -242,6 +242,26 @@ static int uct_iface_is_same_device(const uct_iface_h iface,
     return !memcmp(device_addr, dev_addr, attr.device_addr_len);
 }
 
+void uct_iface_fill_info_str_buf(const uct_iface_is_reachable_params_t *params,
+                                 const char *fmt, ...)
+{
+    char *info_str_buf      = UCS_PARAM_VALUE(UCT_IFACE_IS_REACHABLE_FIELD,
+                                              params, info_string, INFO_STRING,
+                                              NULL);
+    size_t info_str_buf_len = UCS_PARAM_VALUE(UCT_IFACE_IS_REACHABLE_FIELD,
+                                              params, info_string_length,
+                                              INFO_STRING_LENGTH, 0);
+
+    va_list ap;
+
+    va_start(ap, fmt);
+    if ((info_str_buf != NULL) && (info_str_buf[0] == '\0')) {
```

Comment:
why need to check info_str_buf[0] == '\0'?

Reply:
Oh, I mistakenly put it back.